client/my-sites/woocommerce/woop/landing-page.tsx

```diff
@@ -21,25 +20,20 @@ const WarningsOrHoldsSection = styled.div`
 `;
 
 interface Props {
-	startSetup: () => void;
 	siteId: number;
 }
 
 const images = [ { src: Image01 }, { src: Image02 }, { src: Image03 }, { src: Image04 } ];
 
-const WoopLandingPage: React.FunctionComponent< Props > = ( { startSetup, siteId } ) => {
+const WoopLandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 	const { __ } = useI18n();
 	const navigationItems = [ { label: 'WooCommerce', href: `/woocommerce-installation` } ];
 	const ctaRef = useRef( null );
 
 	const { hasBlockers, wpcomDomain, isDataReady } = useWooCommerceOnPlansEligibility( siteId );
 
 	function onCTAClickHandler() {
-		if ( isEnabled( 'woop' ) ) {
-			return page( `/start/woocommerce-install/?site=${ wpcomDomain }` );
-		}
-
-		return startSetup();
+		return page( `/start/woocommerce-install/?site=${ wpcomDomain }` );
```

Comment:
Removing the feature flag 🎉

Reply:
Sorry for being a spoilsport, I don't wanna ruin the party but should we [enable the `woop` flag for production](https://github.com/Automattic/wp-calypso/blob/trunk/config/production.json#L19) instead of removing it now?

Reply:
Thanks, that will make reverting easier if needed.